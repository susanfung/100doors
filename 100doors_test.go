package main

import (
	"testing"
	approvals "github.com/approvals/go-approval-tests"
)

func TestInitialState(t *testing.T) {
	approvals.VerifyArray(t, Answer(0))
}

func TestFirstPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(1))
}

func TestSecondPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(2))
}