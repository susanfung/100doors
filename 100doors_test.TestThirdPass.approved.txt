[0]=opened
[1]=closed
[2]=closed
[3]=closed
[4]=opened
[5]=opened
[6]=opened
[7]=closed
[8]=closed
[9]=closed
[10]=opened
[11]=opened
[12]=opened
[13]=closed
[14]=closed
[15]=closed
[16]=opened
[17]=opened
[18]=opened
[19]=closed
[20]=closed
[21]=closed
[22]=opened
[23]=opened
[24]=opened
[25]=closed
[26]=closed
[27]=closed
[28]=opened
[29]=opened
[30]=opened
[31]=closed
[32]=closed
[33]=closed
[34]=opened
[35]=opened
[36]=opened
[37]=closed
[38]=closed
[39]=closed
[40]=opened
[41]=opened
[42]=opened
[43]=closed
[44]=closed
[45]=closed
[46]=opened
[47]=opened
[48]=opened
[49]=closed
[50]=closed
[51]=closed
[52]=opened
[53]=opened
[54]=opened
[55]=closed
[56]=closed
[57]=closed
[58]=opened
[59]=opened
[60]=opened
[61]=closed
[62]=closed
[63]=closed
[64]=opened
[65]=opened
[66]=opened
[67]=closed
[68]=closed
[69]=closed
[70]=opened
[71]=opened
[72]=opened
[73]=closed
[74]=closed
[75]=closed
[76]=opened
[77]=opened
[78]=opened
[79]=closed
[80]=closed
[81]=closed
[82]=opened
[83]=opened
[84]=opened
[85]=closed
[86]=closed
[87]=closed
[88]=opened
[89]=opened
[90]=opened
[91]=closed
[92]=closed
[93]=closed
[94]=opened
[95]=opened
[96]=opened
[97]=closed
[98]=closed
[99]=closed